students/ravi/Session02/fizz_buzz.py

```diff
@@ -0,0 +1,11 @@
+for i in range(100):
+    if (i+1)%3+(i+1)%5==0:
```

Comment:
You don't need to add +1 here (or anywhere else downstream) if you add +1 to the range function above.